From 7cb434a8a6e7c14480f4649d589792a9b6af1a13 Mon Sep 17 00:00:00 2001
From: hellokamlesh <kamlesh@iitk.ac.in>
Date: Sun, 4 Mar 2018 22:05:16 +0530
Subject: [PATCH] added one line in main

---
 hello.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hello.c b/hello.c
index 19338fe..d56703d 100644
--- a/hello.c
+++ b/hello.c
@@ -4,6 +4,7 @@ void main(){
   printf("HElloword!\n");
   printf("This must be a monolithic design\n");
   microkernel_sendmsg("is more portable");
+  printf("add one line in main\n");
 }
 void microkernel_sendmsg(char *a) {
   printf("microkernel:%s\n",a);
--
2.14.1
